README.md

```diff
@@ -1,149 +1,161 @@
 # analyze-code-security-scc
 
+
 ## Description
 
-This Github action identifies insecure configurations in Infrastructure as Code (IaC) files for Google Cloud resources. 
-This action requires Terraform plan files in JSON format for scanning.
+
+This Github action identifies insecure configurations in Infrastructure as Code (IaC) files for Google Cloud resources. This action requires Terraform plan files in JSON format for scanning.
+
 
 Use this action to detect and remediate issues in IaC files for Google Cloud before you deploy the resources.
 
+
 This action lets you:
 - Scan IaC template files (such as Terraform plan files).
 - Display issues with their severity as a SARIF Report in the GitHub Workspace after a scan completes.
 - Define severity-based failure criteria for passing or failing the build.
 
-> [!NOTE] 
-> This is a Security Command Center Premium tier offering for subscription customers only. You must activate the Security Command Center Premium tier in the Google Cloud organization to use this feature.
 
-**This is not an officially supported Google product, and it is not covered by a
-Google Cloud support contract. To report bugs or request features in a Google
-Cloud product, please contact [Google Cloud
-support](https://cloud.google.com/support).**
+Note: This is a Security Command Center Premium tier offering for subscription customers only. You must activate the Security Command Center Premium tier in the Google Cloud organization to use this feature.
```

Comment:
Id prefer to keep the GitHub [markdown alert](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) syntax. This helps call this out to users.

```markdown
> [!IMPORTANT]
> This is a Security Command Center Premium tier offering for subscription customers only. You must activate the Security Command Center Premium tier in the Google Cloud organization to use this feature.
> 
> This offering is covered by the Pre-GA Offerings Terms of the Google Cloud Terms of Service.

> [!CAUTION]
> Don’t enter any sensitive information such as passwords and other personal identifiable information in the Terraform plan files.
```

Example
> [!IMPORTANT]
> This is a Security Command Center Premium tier offering for subscription customers only. You must activate the Security Command Center Premium tier in the Google Cloud organization to use this feature.
> 
> This offering is covered by the Pre-GA Offerings Terms of the Google Cloud Terms of Service.

> [!CAUTION]
> Don’t enter any sensitive information such as passwords and other personal identifiable information in the Terraform plan files.

Reply:
Done.